src/test/java/library/view/ModeTest.java

```diff
@@ -0,0 +1,37 @@
+package library.view;
+
+import library.repository.BookRepository;
+import library.repository.FileBookRepository;
+import library.repository.InMemoryBookRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ModeTest {
+
+    @Test
+    @DisplayName("모드 코드로 모드를 찾을 수 있어야 합니다.")
+    void testFindByCode() {
+        Optional<Mode> normalMode = Mode.findByCode("1");
+        assertEquals(Mode.NORMAL, normalMode.get());
+
+        Optional<Mode> testMode = Mode.findByCode("2");
+        assertEquals(Mode.TEST, testMode.get());
+
+        Optional<Mode> invalidMode = Mode.findByCode("3");
+        assertFalse(invalidMode.isPresent());
+    }
```

Comment:
```suggestion
    private static Stream<Arguments> modeCode() {
        return Stream.of(
            Arguments.of(Mode.NORMAL, "1"),
            Arguments.of(Mode.TEST, "2")
        );
    }

    @MethodSource("modeCode")
    @ParameterizedTest(name = "모드 {0}의 코드는 {1}입니다.")
    void testFindByCode(Mode mode, String code) {
        Optional<Mode> actual = Mode.findByCode(code);
        assertThat(actual.get()).isEqualTo(mode);
    }
```

테스트가 둘 이상의 문맥이 존재합니다. -> 정상적인 케이스 (1, 2), 비정상적인 케이스(3 ~ 등등 존재하지 않는 입력)
이는 테스트 코드를 분리하여 작성하는 것이 좋습니다.